vnda/loaders/productDetailsPage.ts

```diff
@@ -40,6 +40,7 @@ async function loader(
       const result = await api["GET /api/v2/products/:id"]({
         id,
         include_images: "true",
+        include_inventory_place: "true"
```

Comment:
as it is optional on the API this should be optional on the loader, make it a property